src/main/java/org/scijava/io/location/FileLocation.java

```diff
@@ -118,6 +120,11 @@ public boolean isDirectory() {
 		return file.isDirectory();
 	}
 
+	@Override
+	public String toString() {
+		return "FileLocation: " + file.getAbsolutePath();
```

Comment:
This might be of particular interest when calling modules/scripts headless and handing over parameters as `String`s, no? In that case, I'd vote for URI-style formats to be able to losslessly convert between `String` and `Location` both ways.

Reply:
We already have a mechanism to convert from `String` to `Location` via the `LocationService`. Also every location has a `toURI` method to turn it into an URI which can be turned into an Location. This `toString` override was meant for logging etc.